neqo-transport/src/connection.rs

```diff
@@ -1026,6 +1026,15 @@ impl Connection {
                     self.crypto.states.init(self.role, &packet.dcid());
                 }
                 (PacketType::VersionNegotiation, State::WaitInitial, Role::Client) => {
+                    if let Ok(versions) = packet.supported_versions() {
+                        if versions.contains(&QUIC_VERSION) {
+                            // Ignore VersionNegotiation packets that contain the current version or only
+                            // contain reserved versions. (decode_version_negotiation will remove reserved
+                            // versions)
```

Comment:
Update this comment.

```suggestion
                            // Ignore VersionNegotiation packets that contain the current version.
```